Makefile

```diff
@@ -2,7 +2,7 @@ PIP := python -m pip --disable-pip-version-check
 WEBPACK := yarn build-acceptance
 
 freeze-requirements:
-	@python -S -m tools.freeze_requirements
```

Comment:
It would be great if we tested this target in CI so it doesn't regress. Maybe inside `development-environment.yml`.